faunadb/client.py

```diff
@@ -88,6 +116,8 @@ def __init__(
     self.pool_connections = pool_connections
     self.pool_maxsize = pool_maxsize
 
+    self.last_txn_time = _LastTxnTime()
```

Comment:
When you create a new session client, we should also share this instance.

Reply:
That's a good point and I had meant to think about this - should it be another argument to `new_session_client()`?